HSEARCH-4616 Rename automatic indexing to something more explicit (#3536)
hibernate/hibernate-search: documentation/src/main/asciidoc/public/reference/_coordination.adoc

=== How indexing works without coordination

[[coordination-none-indexing-detected-changes]]

> Changes have to occur in the ORM session in order to be detected for <<listener-triggered-indexing,listener-triggered indexing>>::

Reviewer: ```suggestion
Changes have to occur in the ORM session in order to trigger <<listener-triggered-indexing,indexing listeners>>::
```

The `outbox-polling` strategy implements coordination through
<<coordination-outbox-polling-schema,additional tables>> in the application database.

> <<architecture-hsearch-indexing,Indexing>> is implemented

Reviewer: Not mass indexing, though.

```suggestion
<<architecture-hsearch-indexing,Explicit and listener-triggered indexing>> are implemented
```

=== [[coordination-database-polling-indexing]] How indexing works with `outbox-polling` coordination

[[coordination-outbox-polling-indexing-detected-changes]]

> Changes have to occur in the ORM session in order to be detected for <<listener-triggered-indexing,listener-triggered indexing>>::

Reviewer: ```suggestion
Changes have to occur in the ORM session in order to trigger <<listener-triggered-indexing,indexing listeners>>::
```
